### PR TITLE
Adding a unit test for sensors

### PR DESCRIPTION
**Pull request by JoseM98**
#### Description

Fixes #  

#### Where has this been tested?

  * **Platform(s):** Ubuntu 22.04
  * **Python version(s):** 3.10
  * **Unreal Engine version(s):** UE5 Carla

#### Possible Drawbacks
 
Not tested on Windows.

### CODE REVIEW

**Comment by JoseM98** [comment]
Things ToDo:
- Be able to not print logs when calling to the image save_to_disk(...) method.
- Add a parameter or flag to be able to enable Radar sensor screen debug if needed.

File: PythonAPI/test/API/test_sensor_recording.py [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+                sensor_data[1] = image.frame
+        
+    def destroy_sensors(self):
+        for sensor in self.spawned_sensors:
+            if sensor is not None:
+                sensor.stop()
+                sensor.destroy()
+                
+
+# if __name__ == '__main__':
```
Comment (Blyron) [inline PythonAPI/test/API/test_sensor_recording.py:426]: Remove main function

File: PythonAPI/test/API/test_sensor_recording.py [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+            #['sensor.camera.depth', cc.Depth, 'CameraDepth (GrayScale)', {}],
+            ['sensor.camera.depth', cc.LogarithmicDepth, 'CameraDepth (LogarithmicGrayScale)', {}],
+            #['sensor.camera.semantic_segmentation', cc.Raw, 'CameraSemanticSegmentation (Raw)', {}],
+            ['sensor.camera.semantic_segmentation', cc.CityScapesPalette, 'CameraSemanticSegmentation (CityScapesPalette)', {}],
+            #['sensor.camera.instance_segmentation', cc.Raw, 'CameraInstanceSegmentation (Raw)', {}],
+            ['sensor.camera.instance_segmentation', cc.CityScapesPalette, 'CameraInstanceSegmentation (CityScapesPalette)', {}],
+            ['sensor.lidar.ray_cast', None, 'Lidar (Ray-Cast)', {'range': '50'}],
+            ['sensor.lidar.ray_cast_semantic', None, 'SemanticLidar (Ray-Cast)', {'range': '50'}],
+            #['sensor.camera.dvs', cc.Raw, 'DynamicVisionSensor', {}],
+            #['sensor.camera.rgb', cc.Raw, 'Camera RGB Distorted', {'lens_circle_multiplier': '3.0', 'lens_circle_falloff': '3.0', 'chromatic_aberration_intensity': '0.5', 'chromatic_aberration_offset': '0'}],
```
Comment (Blyron) [inline PythonAPI/test/API/test_sensor_recording.py:345]: Remove commented code or leave comments

File: PythonAPI/test/API/test_sensor_recording_fast.py [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+        
+    def destroy_sensors(self):
+        for sensor in self.spawned_sensors:
+            if sensor is not None:
+                sensor.stop()
+                sensor.destroy()
+                
+
+#if __name__ == '__main__':
+#    unittest.main()
```
Comment (Blyron) [inline PythonAPI/test/API/test_sensor_recording_fast.py:412]: Remove main function 
EOF is missing

**Review by Blyron — APPROVED** [review]

**Pull request merged**